core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -881,6 +884,16 @@ public void withStartupAttempts(int attempts) {
         this.startupAttempts = attempts;
     }
 
+    @Override
+    public void close() {
+        stop();
+    }
+
+    public SELF withCustomizer(Consumer<CreateContainerCmd> hook) {
```

Comment:
No, don't think we need it. My point was only about the naming, including the word _post_ makes the API a bit more self documenting.